crates/uv/tests/it/common/mod.rs

```diff
@@ -606,6 +617,7 @@ impl TestContext {
             .env(EnvVars::COLUMNS, "100")
             .env(EnvVars::PATH, path)
             .env(EnvVars::HOME, self.home_dir.as_os_str())
+            .env(EnvVars::XDG_CONFIG_HOME, self.user_config_dir.as_os_str())
```

Comment:
Should we just set `USERPROFILE` to `self.home_dir` instead? Do we need to set `XDG_CONFIG_HOME` specifically?